go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/docker/docker v20.10.12+incompatible
 	github.com/docker/go-connections v0.4.0
 	github.com/duo-labs/webauthn v0.0.0-20210727191636-9f1b88ef44cc
-	github.com/ethereum/go-ethereum v1.10.18
+	github.com/ethereum/go-ethereum v1.10.19
```

Comment:
Why 19 instead of 20? #6917

Reply:
I think that's just what it gave me as the latest, for some reason?  Just manually changed it to 1.10.20 and still seems to work.